src/main/scala/com/databricks/spark/redshift/Conversions.scala

```diff
@@ -138,3 +138,23 @@ private[redshift] class RedshiftTimestampFormat extends DateFormat {
     }
   }
 }
+
+private[redshift] class RedshiftDateFormat extends DateFormat {
+
+  // Imports and exports with Redshift require that dates are represented
+  // as strings, using the following format
+  private val PATTERN = "yyyy-MM-dd"
+
+  private val redshiftDateFormat = new SimpleDateFormat(PATTERN)
+
+  override def format(
+                       date: Date,
```

Comment:
Minor styling nit: can you indent the parameters by only four spaces relative to the start of the previous line in order to match the style on line 126 in this same file?